test/unit/actions/noting/remove-part-of-note.spec.js

```diff
@@ -0,0 +1,36 @@
+
+var util = require('util');
+var path = require('path');
+var _ = require('lodash');
+var chai = require('chai');
+var expect = chai.expect;
+
+var h = require('virtual-hyperscript');
+var VText = require('vtree/vtext');
+var VFocus = require(path.resolve(process.cwd(), 'src/vfocus'));
+
+var removePartOfNote = require(path.resolve(process.cwd(), 'src/actions/noting/remove-part-of-note'));
+
+describe('removePartNote()', ()=> {
+
+  it('should remove a note segment that is selected', ()=> {
+
+    var tree = h('p', [
+      h('gu-note', {dataset: {noteId: 1234}}, [
+      new VText('this'),
+      new VText('is'),
+      h('em.scribe-marker'),
+      new VText('some'),
+      h('em.scribe-marker'),
+      new VText('text')
+    ])
+    ]);
+
+    tree = new VFocus(tree);
+    tree = removePartOfNote(tree);
+
+    expect(JSON.stringify(tree).match(/gu-note/g).length).to.be.equal(3);
```

Comment:
This checks that _a_ note is removed, not necessarily that it's the selected note that's removed, doesn't it?